### ISSUE TITLE
Call to undefined function mysql_connect() (PHP5.6.35, MYSQL5.7, APACHE2)

### ISSUE THREAD

**Issue by jerfeson** [laradock/laradock#1483]
### Info:
- Docker version Docker version 18.03.0-ce, build 0520e24
- Laradock commit  #0c41fce: 
- System info PC 
- System info 10.0.16299 Compiltaion 16299

### Issue:

According to information the mysql_connect function was deprecated in version 7.0 of PHP, but when I set up the environment with PHP 5.6.35, my projects with this function do not work

### Expected behavior:

The use of this method should occur naturally.
_____

### Reproduce:

`<?php

$link = mysql_connect('localhost', 'mysql_user', 'mysql_password');

if (!$link) {
    die('Could not connect: ' . mysql_error());
}

echo 'Connected successfully';

mysql_close($link);

?>`
_____

```
### PHP Version ###########################################

# Select a PHP version of the Workspace and PHP-FPM containers (Does not apply to HHVM). Accepted values: 7.2 - 7.1 - 7.0 - 5.6 - 5.5
PHP_VERSION=5.6

### PHP Interpreter #######################################

# Select the PHP Interpreter. Accepted values: hhvm - php-fpm
PHP_INTERPRETER=php-fpm

...

```

**Comment by philippeboyd** [comment]
Do you have the extension installed?

`mysql_connect()` is still part of 5.6 (removed in 7)
try to do the following :

```bash
docker exec $container_name docker-php-ext-install mysql
docker restart $container_name
```

**Comment by jerfeson** [comment]
Actually, the extension was not installed, but this should already be automatic at the time of mounting the container, I will study laradock, and try to develop this, for now I am still exploring the tool.


Many thanks for the help;

**Comment by hlorofos** [comment]
Any update how this could be persistently done?

**Comment by vladyslavstartsev** [comment]
@hlorofos you can put the code (that was suggested by @philippeboyd) in your Dockerfile. Or even send PR. The problem is, it probably will not be merged, because the 5.6 EOL is in 2 mouths from now.
So the installing of the extension is mostly up to you.
